_includes/menu.html

```diff
@@ -5,7 +5,7 @@
 		<li>
 			<a
 				href="{{ site.baseurl }}{{ node.url | remove: "/index" }}"
-				{% if page.url == node.url %}class="active"{% endif %}
+				{% if (node.category and node.category == page.category) or page.url == node.url %}class="active"{% endif %}
```

Comment:
Hm. I added `node.category and` because it highlighted multiple pages due to both values being null, but now it doesn't highlight what it should.

Should I split it up into if/elsif, or add `category` for all pages?